tests/test-sources/plugins/by-name/abolish/default.nix

```diff
@@ -0,0 +1,19 @@
+_: {
+  empty = {
+    plugins.abolish = {
+      enable = true;
+    };
+  };
+
+  example = {
+    plugins.abolish = {
+      enable = true;
+    };
+  };
+
+  defaults = {
+    plugins.abolish = {
+      enable = true;
+    };
+  };
```

Comment:
This is redundant in this case. Let's not increase the CI duration for nothing ;)